### PR TITLE
Update built-in-monitoring.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/built-in-monitoring.md [hunk 1]
```diff
@@ -19,7 +19,11 @@ To view the metrics on the Monitoring page, take the following steps:
 
 2. Click the **Monitoring** tab.
 
-## Monitoring metrics
+## Metrics Retention Policy
+
+All monitoring data are kept for 15 days.
```
Comment (lilyjazz) [inline tidb-cloud/built-in-monitoring.md:24]:
```suggestion
All monitoring data are kept for 2 days.
```

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/built-in-monitoring.md [hunk 2]
```diff
@@ -19,7 +19,11 @@ To view the metrics on the Monitoring page, take the following steps:
 
 2. Click the **Monitoring** tab.
 
-## Monitoring metrics
+## Metrics Retention Policy
+
+All monitoring data are kept for 2 days.
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:24]:
```suggestion
All monitoring data is kept for 2 days.
```

File: tidb-cloud/built-in-monitoring.md [hunk 3]
```diff
@@ -19,7 +19,11 @@ To view the metrics on the Monitoring page, take the following steps:
 
 2. Click the **Monitoring** tab.
 
-## Monitoring metrics
+## Metrics Retention Policy
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:22]:
```suggestion
## Metrics retention policy
```

File: tidb-cloud/built-in-monitoring.md [hunk 4]
```diff
@@ -19,7 +19,11 @@ To view the metrics on the Monitoring page, take the following steps:
...
+
+All monitoring data are kept for 2 days.
+
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:25]: Question: How to view yesterday's monitoring data?

Comment (lilyjazz) [reply tidb-cloud/built-in-monitoring.md:25]: change the time picker, users will see the time picker on UI. It's obvious.

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/built-in-monitoring.md [hunk 5]
```diff
@@ -19,7 +19,11 @@ To view the metrics on the Monitoring page, take the following steps:
 
 2. Click the **Monitoring** tab.
 
-## Monitoring metrics
+## Metrics retention policy
+
+All monitoring data is kept for 2 days.
+
+## Monitoring Metrics
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:26]:
```suggestion
## Monitoring metrics
```

**Pull request merged**